src/components/drag-and-drop/draggable-context/draggable-context.js

```diff
@@ -132,11 +207,17 @@ class DraggableContext extends React.Component {
   /**
    * A callback for when dragging begins.
    *
+   * Sets state of dragging to true which
+   * will trigger the start of auto scroll
+   * if it is enabled.
+   *
    * @method handleBeginDrag
    * @param {Object} props
    * @return {Void}
    */
   handleBeginDrag = (props) => {
+    this.setState({ dragging: true });
```

Comment:
instead of `state.dragging`, does `state.activeIndex` also not determine if drag is in progress or not?

Reply:
be careful though that index could be `0` which is falsey in JavaScript.